Index: src/main/java/veiw/RegistrazioneCtrlGrafico.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/veiw/RegistrazioneCtrlGrafico.java b/src/main/java/view/RegistrazioneCtrlGrafico.java
rename from src/main/java/veiw/RegistrazioneCtrlGrafico.java
rename to src/main/java/view/RegistrazioneCtrlGrafico.java
--- a/src/main/java/veiw/RegistrazioneCtrlGrafico.java	
+++ b/src/main/java/view/RegistrazioneCtrlGrafico.java	
@@ -1,6 +1,7 @@
-package veiw;
+package view;
 
 import controllerApplicativo.RegistrazioneCtrlApplicativo;
+import engineering.UserBean;
 import javafx.fxml.FXML;
 import javafx.fxml.FXMLLoader;
 import javafx.scene.Scene;
@@ -64,7 +65,7 @@
         }
     }
 
-    private void getData(){
+    private UserBean getData2(){
         //controlla se ha inserito davvero qualcosa
         user_name = name.getText();
         user_email = email.getText();
@@ -72,6 +73,19 @@
         /*password in chiaro*/
         user_password = password.getText();
         user_conf_pw = conf_password.getText();
+
+        if(user_name.isEmpty() || user_email.isEmpty() || user_password.isEmpty() || user_conf_pw.isEmpty()) {
+            error_pw.setText("CAMPI VUOTI");
+            error_pw.setVisible(true);
+            return null;
+        } else {
+            UserBean userBeanInfo = new UserBean();
+            userBeanInfo.setUserEmail(user_email);
+            userBeanInfo.setName(user_name);
+            userBeanInfo.setPass(user_password);
+            return userBeanInfo;
+        }
+
     }
 
 }
Index: src/main/java/veiw/AddPlaylistCtrlGrafico.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/veiw/AddPlaylistCtrlGrafico.java b/src/main/java/view/AddPlaylistCtrlGrafico.java
rename from src/main/java/veiw/AddPlaylistCtrlGrafico.java
rename to src/main/java/view/AddPlaylistCtrlGrafico.java
--- a/src/main/java/veiw/AddPlaylistCtrlGrafico.java	
+++ b/src/main/java/view/AddPlaylistCtrlGrafico.java	
@@ -1,4 +1,4 @@
-package veiw;
+package view;
 
 public class AddPlaylistCtrlGrafico {
 }
Index: src/main/java/veiw/HomePageCtrlGrafico.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/veiw/HomePageCtrlGrafico.java b/src/main/java/view/HomePageCtrlGrafico.java
rename from src/main/java/veiw/HomePageCtrlGrafico.java
rename to src/main/java/view/HomePageCtrlGrafico.java
--- a/src/main/java/veiw/HomePageCtrlGrafico.java	
+++ b/src/main/java/view/HomePageCtrlGrafico.java	
@@ -1,4 +1,4 @@
-package veiw;
+package view;
 
 import javafx.fxml.FXMLLoader;
 import javafx.scene.Scene;
Index: src/main/java/veiw/LoginCtrlGrafico.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/veiw/LoginCtrlGrafico.java b/src/main/java/view/LoginCtrlGrafico.java
rename from src/main/java/veiw/LoginCtrlGrafico.java
rename to src/main/java/view/LoginCtrlGrafico.java
--- a/src/main/java/veiw/LoginCtrlGrafico.java	
+++ b/src/main/java/view/LoginCtrlGrafico.java	
@@ -1,4 +1,4 @@
-package veiw;
+package view;
 
 import controllerApplicativo.LoginCtrlApplicativo;
 import engineering.LoginBean;
